xml/geo_manage_i.xml

```diff
@@ -164,25 +197,115 @@ booth[27900]: 2014/08/13_10:21:23 info: revoke succeeded!</screen>
      stopped or demoted cleanly.
     </para>
    </step>
-   <step>
+  <!-- <step>
     <para>
      Revoke <literal>&ticket1;</literal> from site
      <literal>&cluster1;</literal> with:
     </para>
 <screen>&prompt.root;<command>booth</command> revoke -s 192.168.201.100 &ticket1;</screen>
-   </step>
+   </step>-->
    <step>
     <para>
-     After the ticket has been revoked from its original site, grant it to the
-     site <literal>&cluster2;</literal> with:
+     <!--After the ticket has been revoked from its original site, -->Grant
+     <literal>&ticket1;</literal> to the site <literal>&cluster2;</literal> with:
     </para>
 <screen>&prompt.root;<command>booth</command> grant -s 192.168.202.100 &ticket1;</screen>
    </step>
+   <step>
+    <para>
+     Activate <literal>&ticket1;</literal> on site <literal>&cluster1;</literal> with:
+    </para>
+<screen>&prompt.root;<command>booth</command> grant -a 192.168.202.100 &ticket1;</screen>
```

Comment:
This needs to be done by "crm_ticket -t ticket1 -a" in cluster1.

Reply:
Thanks, changed.